packages/web/src/pages/sign-up-page/components/FollowArtistTile.tsx

```diff
@@ -30,6 +35,52 @@ const FollowArtistTile = (props: FollowArtistTileProps) => {
   const coverPhoto = useCoverPhoto(user_id, WidthSizes.SIZE_640)
   const [followField] = useField({ name: 'selectedArtists', type: 'checkbox' })
 
+  const {
+    togglePreview,
+    nowPlayingArtistId,
+    isPlaying: isPreviewPlaying
+  } = useContext(SelectArtistsPreviewContext)
+
+  const isPlaying = isPreviewPlaying && nowPlayingArtistId === user_id
+
+  const [avatar] = useHover((isHovered) => (
+    <Box w={72} h={72} css={{ position: 'absolute', top: 34 }}>
+      <Flex
+        h={74}
+        w={74}
+        justifyContent='center'
+        alignItems='center'
+        css={{
+          visibility: isHovered || isPlaying ? 'visible' : 'hidden',
+          pointerEvents: 'none',
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          borderRadius: 100,
+          opacity: 0.75,
+          background:
+            'radial-gradient(50% 50% at 50% 50%, rgba(0, 0, 0, 0.50) 0%, rgba(0, 0, 0, 0.10) 100%)',
+          zIndex: 2
```

Comment:
nit: I wonder if we're taking the inline css a little too far here 😅, this is about ~13 style rules and not just "quick one-off" things. IMO this is still a good use case for a style file but maybe thats just me

Reply:
Good point. Wondering what the meta is here post-harmony. I'm down for either, and please feel free to push changes to this if you see fit